### DISCUSSION TITLE
Quality of generated data : mxstep error

### DISCUSSION THREAD

**Discussion by Ansh-Lodhi** [pybamm-team/PyBaMM#3010] [Q&A] [answered]
I am running a DFN simulation with SEI turned on. The protocol involves cycling between 4.2V to 2.7V with Okane chemistry at 1C (5A). 
I continuously get errors along the following lines :
 
At t = 472.897 and h = 6.92889e-15, the corrector convergence failed repeatedly or with |h| = hmin.
At t = 232.897, , mxstep steps taken before reaching tout.

I want to understand extent of effect this may have on the generated cycling data. I usually ignored these errors.

Thank You

**Comment by brosaplanella** [comment, marked as answer]
If the simulation continues and produces a result then there is no impact on the solution (the solver takes multiple attempts and some may no work but the overall solver does not crash). If it crashes and doesn't provide a solution, then you will have to play with the tolerances of the solver probably.
